### PR TITLE
chore: Examples pruning part 2

### PR DESCRIPTION
**Pull request by MikhailKardash**
## Description

Move rest of examples pruning examples to legacy.
Remove tutorials and model hub v1 examples docs.

## Test Plan

CI Passes

## Commentary (optional)

Guideline for creating and maintaining examples still in progress.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket
MLG-904

### CODE REVIEW

File: docs/tutorials/index.rst [hunk 1]
```diff
@@ -9,27 +9,8 @@
...
-+---------------------------------+--------------------------------------------------------------+
-| :doc:`pytorch-porting-tutorial` | The PyTorch porting tutorial provides helpful hints to       |
-|                                 | successfully integrate PyTorch code with the Determined      |
-|                                 | PyTorchTrial API.                                            |
-+---------------------------------+--------------------------------------------------------------+
-| :doc:`tf-mnist-tutorial`        | The TensorFlow Keras Fashion MNIST tutorial describes how to |
-|                                 | port a ``tf.keras`` model to Determined.                     |
-+---------------------------------+--------------------------------------------------------------+
+To get started with a :ref:`Trial API <high-level-apis>`, visit the :ref:`pytorch-mnist-tutorial`.
+This tutorial shows you how to port a simple image classification model for the MNIST dataset.
```
Comment (tara-det-ai) [inline docs/tutorials/index.rst:13]: we'll need both sections: Quickstart and Get Started with a Trial API.

`###########
 Tutorials
###########

.. meta::
   :description: Choose a tutorial to help you get started training machine learning models. You'll find beginner level and more advanced tutorials with links to user guides and examples.

************
 Quickstart
************

To get started with your first experiment, visit the :ref:`Quickstart for Model Developers
<qs-mdldev>`.

******************************
 Get Started with a Trial API
******************************

To get started with a :ref:`Trial API <high-level-apis>`, visit the :ref:`pytorch-mnist-tutorial`.
This tutorial shows you how to port a simple image classification model for the MNIST dataset.

**********************************************
 Want to Learn About a Specific Training API?
**********************************************

:ref:`Training API Guides <apis-howto-overview>` describe how to take your existing model code and
train your model in Determined.

****************
 Try an Example
****************

Examples let you build off of an existing model that already runs on Determined. Visit our
:ref:`Examples <example-solutions>` to see if the model you'd like to train is already available.

.. _pytorch mnist example: https://github.com/PyTorch/examples/blob/master/mnist/main.py

.. toctree::
   :hidden:

   Run Your First Experiment <pytorch-mnist-local-qs>
   PyTorch MNIST Tutorial <pytorch-mnist-tutorial>
   `

**Review by tara-det-ai — APPROVED** [review]
docs pages LGTM

**Comment by MikhailKardash** [comment]
Made a new PR for this: https://github.com/determined-ai/determined/pull/7918
